code/modules/medical/genetics/geneticsItems.dm

```diff
@@ -189,3 +175,78 @@
 		else
 			boutput(user, "<span class='alert'>You stab [M], but nothing happens.</span>")
 		return
+
+/obj/item/dna_scrambler
+	name = "dna scrambler"
+	desc = "An illegal retroviral genetic serum designed to randomize the user's identity, store it, and apply it later."
+	icon = 'icons/obj/syringe.dmi'
+	inhand_image_icon = 'icons/mob/inhand/hand_medical.dmi'
+	item_state = "syringe_0"
+	icon_state = "dna_scrambler_1"
+	force = 3
+	throwforce = 3
+	w_class = W_CLASS_SMALL
+	var/use_mode = "copy"
```

Comment:
"copy", "paste", and "depleted" should probably be macros.
Just stick this above the type definition:
```
#define SCRAMBLER_MODE_COPY "copy"
#define SCRAMBLER_MODE_PASTE "paste"
#define SCRAMBLER_MODE_DEPLETED "depleted"
```
and this at the end (around line 252):

```
#undef SCRAMBLER_MODE_COPY
#undef SCRAMBLER_MODE_PASTE
#undef SCRAMBLER_MODE_DEPLETED
```
The defines just make it a bit easier to remember and put all the possibilities in the same place, and the `undef` stuff prevents those defines from being used elsewhere (since they never need to be used elsewhere).

Then you'd replace the existing usage, i.e. this line would become `var/use_mode = SCRAMBLER_MODE_COPY`